### DISCUSSION TITLE
How do I associate OIDC users with already-existing users in LibreChat?

### DISCUSSION THREAD

**Discussion by rcdailey** [danny-avila/LibreChat#2676] [Q&A] [answered]
I started using LibreChat without OIDC at first. I already have a local account I've been using. I started plugging it into my Authentik service recently and was able to successfully log in using it. However, I'm not sure how to associate that account with my existing local account. I combed the docs pretty thoroughly and didn't notice anything to help me with this.

How do I associate OIDC/social accounts to local accounts in LibreChat?

**Comment by danny-avila** [comment, marked as answer]
they will be automatically associated if they are using the same email, no need for migration
